Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -5777,7 +5777,7 @@ RefPtr<CSSStyleValue> Parser::parse_font_language_override_value(TokenStream<Com
         }
         transaction.commit();
         if (length < 4)
-            return StringStyleValue::create(MUST(String::formatted("{<4}", string_value)));
+            return StringStyleValue::create(MUST(String::formatted("{:4}", string_value)));
```

Comment:
Don't we still want the `<`?
> Shorter strings are right-padded with spaces

The `<` is what tells formatters to right-pad instead of left-pad, so that'd be:
```c++
return StringStyleValue::create(MUST(String::formatted("{:<4}", string_value)));
```

Reply:
My understanding was that, without the specifier, the default alignment will be used, which gives the value right-padding. I've added the `<` back in though, as that makes it clear that the value is intended to be left-aligned / right-padded.